cdm/core/src/test/java/ucar/nc2/dataset/TestScaleOffsetMissingUnsigned.java

```diff
@@ -286,7 +286,8 @@ public void testScaleOffsetMissingUnsigned() throws URISyntaxException, IOExcept
 
   // This test demonstrated the bug in https://github.com/Unidata/netcdf-java/issues/572, but for unsigned variables.
   @Test
-  public void testNegativeScaleOffsetValidRangeUnsigned() throws URISyntaxException, IOException {
+  public void testNegativeScaleOffsetValidRangeUnsigned()
+      throws URISyntaxException, IOException, InvalidRangeException {
```

Comment:
Don't think this change is still needed
```suggestion
  public void testNegativeScaleOffsetValidRangeUnsigned() throws URISyntaxException, IOException {
```